upf/pfcp.h

```diff
@@ -2329,7 +2384,7 @@ typedef struct
     pfcp_user_plane_ip_resource_information_t
     * user_plane_ip_resource_information;
   pfcp_tp_build_id_t tp_build_id;
-  pfcp_ue_ip_address_pool_identity_t *ue_ip_address_pool_identity;
+  pfcp_ue_ip_address_pool_identity_t ue_ip_address_pool_identity;
```

Comment:
* this is wrong, should be `UE IP address Pool Information`
* please keep the fields in the order from the specification and 3GPP IEs before the vendor IEs

Reply:
fixed